### PR TITLE
Membership Proposal - Selene Xenia

### PR DESCRIPTION
**Pull request by Salynne**
I've attached my membership proposal in a pull request as requested. It's in markdown, if you prefer a different format please let me know. I'm on the discord, feel free to ask me questions there or here.

### CODE REVIEW

**Review by pedroMMM — COMMENTED** [review]
🥳  the 1st membership proposal! 

@Salynne I liked what you had to say and I think you are grounded in reality!

Of course, the final decision lies with @kris-nova but I think @Salynne is a great addition to the team 💜

File: proposals/selene_xenia.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+I am the type of person who does not historically seek power but is granted it by being helpful in communities, usually made administrator or moderator of a community, and ultimately am now a people manager in tech. This may be the first time I have ever formally asked up front for some form of control or power to a new group of people, outside of tech job interviews. Writing this and selling myself is uncomfortable but I take this seriously enough that I'm doing it anyway.
+
+
+## Goals
+
+As a Member, I set the following goals for myself:
+* Sponsor 2 candidates simultaneously on average. At this stage it's unclear how long in calendar time a sponsorship takes, or how much time it will take per person. It is also likely unique to the situation. So I would set a goal on how many different people I can feel comfortable being available to mentor at nearly the drop of a hat at any given time. This is in addition to the folks I mentor outside the organization. If they became sponsors
```
[the remark this answers is not in the document]
Comment (Salynne) [reply proposals/selene_xenia.md:15]:
```suggestion
* Sponsor 2 candidates simultaneously on average. At this stage it's unclear how long in calendar time a sponsorship takes, or how much time it will take per person. It is also likely unique to the situation. So I would set a goal on how many different people I can feel comfortable being available to mentor at nearly the drop of a hat at any given time. This is in addition to the folks I mentor outside the organization. If they became sponsors, this goal would be adjusted to account for new people.
```

I got distracted writing the rest of my thought about goals and never came back haha

[the remark this answers is not in the document]
Comment (pedroMMM) [reply proposals/selene_xenia.md:15]: It happens

File: proposals/selene_xenia.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* Work on raising money, to get a baseline for what I can accomplish, and set a goal for continued fund raising off that. I will be doing this regardless of acceptance of Membership.
```
[the remark this answers is not in the document]
Comment (kris-nova) [reply proposals/selene_xenia.md:16]: in my mind i think goals are a personal thing - the foundation doesn't dictate what the goals should be - we just think each person should some for themselves so we can see how we are doing over time

[the remark this answers is not in the document]
Comment (kris-nova) [reply proposals/selene_xenia.md:16]: the foundation is ultimately here as a building block - a platform - a kernel - for others to use 

we have some legal considerations to keep in mind as we grow - but otherwise the sky is the limit

in theory if we had someone passionate about things like stores, events, books, workshops, campaigns, etc we have a platform in which we could enable that 

tldr; members have 2 main responsibilities -- but that doesnt mean members can't do anything else :)

File: proposals/selene_xenia.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* 99% availability for voting. I cannot truly guarantee 100% but I will get as close as can, with likely planned and unplanned outages.
+
+As a Member, I expect to contribute to the organization of the organization. That means helping other Members, including with how to help them with their candidates in addition to mine. That means writing content for the website where appropriate. That means making suggestions to improve things, and always be thinking of how to do even better. I do not set formal objective goals for this work - it is organic work that is done to build something that works well with people. It can mean introducing new formal objectively measurable however.
```
Comment (kris-nova) [inline proposals/selene_xenia.md:19]: > As a Member, I expect to contribute to the organization of the organization.

This is a great sentence. We might consider making this a membership value or something in the future.

File: proposals/selene_xenia.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pedroMMM) [reply proposals/selene_xenia.md:16]: Ok, that tracks. I have some follow up questions about the other different things members can do, but I will keep it on Discord so we can focus on @Salynne

File: proposals/selene_xenia.md (hunk 3 above)
Comment (pedroMMM) [reply proposals/selene_xenia.md:19]: Good catch! I would put it forward and center in https://privilegeescalation.org/membership/

**Review by pedroMMM — APPROVED** [review]

**Comment by kris-nova** [comment]
Welcome to the foundation @Salynne 🎉 

Official 1st member ✨
